src/models/user.js

```diff
@@ -0,0 +1,46 @@
+const { DataTypes, UUIDV4 } = require('sequelize');
+const yup = require('yup');
+
+const { email } = require('../utils/validation/email.js');
+const { password } = require('../utils/validation/password.js');
+const { sequelize } = require('../utils/db.js');
+
+const User = sequelize.define(
+  'User',
+  {
+    id: {
+      type: DataTypes.UUID,
+      defaultValue: UUIDV4,
+      primaryKey: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+    },
+    email: {
+      type: DataTypes.STRING,
+      unique: true,
+      allowNull: false,
+    },
+    password: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    activationToken: {
+      type: DataTypes.STRING,
+    },
+    resetToken: {
+      type: DataTypes.STRING,
+    },
+  },
+  {
+    tableName: 'users',
+  },
+);
+
+module.exports = {
+  User,
+  userSchema: yup.object().shape({
+    email,
+    password,
+  }),
```

Comment:
Ensure that the `email` and `password` fields in the Yup schema are correctly defined and imported from the validation files. If these are Yup schemas, they should be validated as such, e.g., `email: email.required()`, `password: password.required()`.